### ISSUE TITLE
[PSR-12] blank line before namespace

### ISSUE THREAD

**Issue by maxmirazh33** [php-fig/fig-standards#735]
Maybe need add rule for blank line before namespace when not presented declare statements?
Now PSR-2 compatible packages use different variants.

For example, code from yii2 app:
```
<?php
namespace common\models;
//other code
```
and code from laravel app:
```
<?php

namespace App;
//other code
```

I propose in PSR-12 apply one of these variants.

**Comment by michaelcullum** [comment]
This is included in #730

**Comment by samdark** [comment]
Moved to https://github.com/cs-extended/fig-standards/issues/28
